velox/expression/tests/ArgumentTypeFuzzerTest.cpp

```diff
@@ -326,4 +326,79 @@ TEST_F(ArgumentTypeFuzzerTest, unconstrainedSignatureTemplate) {
   ASSERT_EQ(argumentTypes[0]->childAt(0), argumentTypes[1]);
 }
 
+TEST_F(ArgumentTypeFuzzerTest, orderableConstraint) {
+  {
+    for (int i = 0; i < 100; ++i) {
+      auto signature = exec::FunctionSignatureBuilder()
+                           .orderableTypeVariable("T")
+                           .returnType("bigint")
+                           .argumentType("T")
+                           .build();
+      std::mt19937 seed{123456};
+      ArgumentTypeFuzzer fuzzer{*signature, nullptr, seed};
```

Comment:
What's the point of running this code 100 times? Since the seed is fixed each iteration will be exactly same, no? Am I missing something?

BTW, I ran this tests without any changes and it is not failing...

Reply:
Maybe change this something like

```
  {
    auto signature = exec::FunctionSignatureBuilder()
                         .orderableTypeVariable("T")
                         .returnType("bigint")
                         .argumentType("T")
                         .build();

    for (size_t i = 0; i < 100; ++i) {
      std::mt19937 rng(i);
      ArgumentTypeFuzzer fuzzer{*signature, nullptr, rng};
      fuzzer.fuzzArgumentTypes(kMaxVariadicArgs);
      ASSERT_TRUE(fuzzer.argumentTypes()[0]->isOrderable())
          << fuzzer.argumentTypes()[0]->toString();
    }
  }
```

Reply:
Got it, thanks.

Reply:
@mbasmanova Hi Masha, I've updated here according to your guidance, and it fails without any changes.